bofire/utils/torch_tools.py

```diff
@@ -34,7 +34,7 @@
 
 def get_linear_constraints(
     domain: Domain,
-    constraint: Union[LinearEqualityConstraint, LinearInequalityConstraint],
+    constraint: Union[Type[LinearEqualityConstraint], Type[LinearInequalityConstraint]],
     unit_scaled: bool = False,
 ) -> List[Tuple[Tensor, Tensor, float]]:
     """Converts linear constraints to the form required by BoTorch.
```

Comment:
Again, please remove out-dated types from comments.

Reply:
I've updated the signature, but please see other comment about keeping types in docstrings.